### PR TITLE
Subdomain optimization

### PR DESCRIPTION
**Pull request by Idkaros**
Resuming the URL matching and including

### CODE REVIEW

File: ingress_export.js [hunk 1]
```diff
@@ -11,6 +11,8 @@
 // @include http://*ingress.com/intel*
 // @match https://*ingress.com/intel*
 // @match http://*ingress.com/intel*
+// @match https://inter.ingress.com/*
```
Comment (Zetaphor) [inline ingress_export.js]: Is this not covered already by the \*ingress.com/intel\* rules?

**Review by Zetaphor — COMMENTED** [review]
Is this not covered by the

File: ingress_export.js (hunk 1 above)
Comment (Idkaros) [reply ingress_export.js]: No, cause now you can write de url without de */intel* part, and when that happens de plugin doesn't ... ¿function?

Comment (Idkaros) [reply ingress_export.js]: And it must be "intel" ... no "inter"

**Pull request merged**